### ISSUE TITLE
Principle of coordination

### ISSUE THREAD

**Issue by adrik29** [SpeciesFileGroup/taxonworks#2310]
I need help here:

(1) Lawrence (1931) created Neopilioninae [subfamily of Phalangiidae]. This is TW NameId: 322362. ZooBank ...0273.
https://sfg.taxonworks.org/tasks/nomenclature/browse?taxon_name_id=322362

(2) Kauri (1961) elevated Neopilioninae to family Neopilionidae.  This is the same TW NameId: 322362, only elevated to a new rank.

(3) Šilhavý 1970 divided Neopilionidae into 3 subfamilies, including Neopilioninae [subfamily of Neopilionidae]. But this is now another taxon, TW NameId: 322450, which is not yet in ZooBank.
https://sfg.taxonworks.org/tasks/nomenclature/browse?id=322450&taxon_name_id=322450

By the pr. of coordination both are deemed to have been created simultaneously by Lawrence (1931).
Both taxa -362 and -350 at some time were called Neopilioninae.
I need to indicate that -362 was originally called Neopilioninae but it is now Neopilionidae.
While the "True" Neopilioninae is taxon -450
How do I convey this in our system?

In my paper catalogue I can write whatever I wish, but not in TW:
![image](https://user-images.githubusercontent.com/46284943/121821891-4754a600-cc72-11eb-97e6-31cdab79811b.png)
![image](https://user-images.githubusercontent.com/46284943/121821896-53406800-cc72-11eb-82ad-b1a6144e7657.png)

**Comment by adrik29** [comment]
I know that in the ICZN both Neopilioninae are just two coordinated taxa.
We have 
Neopilioninae1 = original spelling of Neopilionidae
Neopilioninae2 = a subfamily of Neopilionidae
But technically, in TW they are **HOMONYMS because they are two different taxa that bear the same name.**

**Comment by proceps** [comment]
@adrik29 Yes, we know that different catalogues has different conventions. In TW all taxon name histery for coordinate name default to lowest rank taxon. In your case it is subfamily. TaxonWorks recognizes coordinate names, and those are not considered as homonyms, as long as one is a direct subordinate of another. On the other hand TW enforses the validity rules for coordinate taxa, for example two coordinate names should have the same author, year and type. If this information is provided for one of the names, it could automatically be populated for the second name, using soft_validation fixes. 
As I said, the history of coordinated names is defaulted to the lower rank. What it means: you can of cause enter the original form of the family group name, and subsequent forms of this name and link those to a taxon of any rank, but when soft_validation fixes are applied, all those relationship will be moved to the lowes rank taxon. This pattern applicable to all forms of relationships. For example, you synonimized one of coordinated names, the relationship will move to the lower rank. You defined a genus as original genus for species, the relationship will move to a coordinate subgenus of this genus. 
Besides, TW will also create a coordinate sub- for you, if a taxon is subdivided into several sub- taxa, but coordinate one does not exists in the DB.
So to summarize, In TW you have several mechanisms to store the history of the name. For family-group name you can provide original spelling and history of subsequent spellings. For each form you can also provide "classified as" relationship, to indicate that this taxon was placed in a particular order, family, subfamily, etc.

**Comment by adrik29** [comment]
@proceps = Thanks, Dmitry, for the lengthy explanation. I will try to convey all of this because it is important (to us anyway). And I already notice that ZooBank treats this is a different way. So I should adapt my thinking AND my entering data to the different systems. In the future I will send you a few examples in my TW project just to see if they look alright!

**Comment by proceps** [comment]
@adrik29 Should we close this for now?

**Comment by adrik29** [comment]
@proceps = yes, you can close it. If needed I'll come again...

**Comment by adrik29** [comment]
I have created a new taxon name called Neopilioninae, and in the module "status" I have indicated that this is the original form (original rank) of Neopilionidae, and NOT the coordinate subfamily Neopilioninae.
Does it make sense?
https://sfg.taxonworks.org/tasks/nomenclature/browse?id=322362&taxon_name_id=322362
![image](https://user-images.githubusercontent.com/46284943/121965875-1a70c380-cd44-11eb-876e-814cd924078d.png)

**Comment by adrik29** [comment]
@proceps = I still need help. I'm getting a flurry of error messages, although I think I entered the right things.
Can you please tell me what I did wrong???
https://sfg.taxonworks.org/tasks/nomenclature/browse?taxon_name_id=322362
![image](https://user-images.githubusercontent.com/46284943/121967193-50af4280-cd46-11eb-9a52-f0a62d466aae.png)
![image](https://user-images.githubusercontent.com/46284943/121967226-615fb880-cd46-11eb-9b05-39098e735e02.png)

**Comment by proceps** [comment]
You do not need "valid" status. By defaut all protonyms are valid. The last relationship in your list require a new protonym. Think about new combination for species. A subsequent combination could be identical with current combination, but this is a separate taxon. 
In your page you would need 3 protonys: Neopilionidae (which is current name); Neopilionidae (with the "original" relationship linking this to the first protonym); Neopilionidae (with the "family form" relationship linking it to the first protonym).

**Comment by adrik29** [comment]
@proceps = Aaaahn -- thanks a lot for the analysis and the solutions!

**Comment by adrik29** [comment]
Ha!

![image](https://user-images.githubusercontent.com/46284943/121972677-0e8bfe00-cd52-11eb-9332-0e08329ae315.png)

**Comment by proceps** [comment]
Yes, the name validation is off for original and subsequent forms. One note, the 3rd protonym "family group name form" requires a citation different from original: Kauri, 1961: 141. Because Lawrence is not the publication where this name was introduced.

**Comment by adrik29** [comment]
@proceps = Like this?

![image](https://user-images.githubusercontent.com/46284943/121973871-983ccb00-cd54-11eb-9869-79bc56946202.png)
